spec/models/category_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
Should this be here?
`require 'rails_helper'`

Reply:
Indeed it should not, it snuck in with a generator. I've removed it, good eye.